components/api/api-cache/pom.xml

```diff
@@ -0,0 +1,50 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.eclipse.dirigible</groupId>
+		<artifactId>dirigible-components-parent</artifactId>
+		<version>11.0.0-SNAPSHOT</version>
+		<relativePath>../../pom.xml</relativePath>
+	</parent>
+
+	<name>Components - API - Cache</name>
+	<artifactId>dirigible-components-api-cache</artifactId>
```

Comment:
I think the `dirigible-components-api-cache` module, should be added as dependency in one of those, in order to be included in the executable JAR:
- https://github.com/eclipse/dirigible/blob/master/components/group/group-api/pom.xml
- https://github.com/eclipse/dirigible/blob/master/components/group/group-api-core/pom.xml
- https://github.com/eclipse/dirigible/blob/master/components/group/group-api-core/pom.xml